backend/service/k8s/cache.go

```diff
@@ -85,6 +91,59 @@ func (s *svc) startInformers(ctx context.Context, cs ContextClientset) {
 	s.topologyInformerLock.Release(topologyInformerLockId)
 }
 
+// cacheFullRelist will list all resources and push them to the topology cache for processing.
+// The importance of doing a semi frequent full re-list give us better cache accuracy,
+// while also keeping resources that are infrequently updated from being cleaned up by the cache TTL.
+// This works in tandem with the LightweightInformers above.
+//
+// Notably we intentionally run these in serial, not only can this cause memory pressure but
+// also being mindful of the kubernetes api servers to reduce burst load.
+func (s *svc) cacheFullRelist(ctx context.Context, lwPods, lwDeployments, lwHPA *cache.ListWatch) {
+	ticker := time.NewTicker(time.Hour * 1)
```

Comment:
hardcoding this here is not great because if the cache TTL dips below this value, because it's configurable or otherwise modified, then we're going to have the same problem.

we need a way to get the TTL value from cache and then use some lesser value here.

Reply:
Yes true this is a problem, I think ill just do 50% of the topocache ttl time. If the ttl is 60min then will refresh every 30min, how does that sound?

Reply:
Actually the TTL for the cache is not yet honored i created a ticket to follow up with a fix for that and will include this in the todo. 

https://github.com/lyft/clutch/issues/1081

Reply:
can we go ahead and fix that ticket first? it should be a pretty quick fix

Reply:
Yeah I can do that, just threading through the configuration to other services might be interesting unless I change the cacheable topology interface will see what i do on the other pr. Other than that it's straight forward.